etl/src/main/java/org/pih/hivmigration/etl/sql/ProgramMigrator.groovy

```diff
@@ -231,7 +231,7 @@ class ProgramMigrator extends SqlMigrator {
         ''')
 
         executeMysql("Load initial enrollments into hivmigration_programs table", '''
-            SET @outcome_transferred_out = (select concept_id from concept where uuid = '3cdd5c02-26fe-102b-80cb-0017a47871b2');
+            SET @outcome_transfer_to_another_ZL_site = (select concept_id from concept where uuid = '3c392f4d-9fdd-44ad-99db-d2bad176f974');
```

Comment:
Took this from the...  "Add enrollment_date and outcome" block earlier on as that appear to be the updated concept for "transfer to another ZL site"...